ghost/core/core/server/api/endpoints/links.js

```diff
@@ -21,5 +21,33 @@ module.exports = {
                 }
             };
         }
+    },
+    bulkEdit: {
+        statusCode: 200,
+        headers: {},
+        options: [
+            'filter'
+        ],
+        data: [
+            'action',
+            'meta'
+        ],
+        validation: {
+            data: {
+                action: {
+                    required: true,
+                    values: ['updateLink']
+                }
+            },
+            options: {
+                filter: {
+                    required: true
+                }
+            }
+        },
+        permissions: false,
```

Comment:
I don't think we should be disabling perms here?

Reply:
D'oh! No we shouldn't, had copied it over from `browse` and missed! I am working on separate PR to add permissions for links endpoint and will then update here as well.

Reply:
Updated permissions and fixed for `browse` endpoint as well with a migration on `main`.